bundles/org.openhab.binding.yeelight/src/main/java/org/openhab/binding/yeelight/internal/handler/YeelightCeilingWithNightHandler.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.yeelight.internal.handler;
+
+import org.eclipse.smarthome.core.library.types.OnOffType;
+import org.eclipse.smarthome.core.thing.ChannelUID;
+import org.eclipse.smarthome.core.thing.Thing;
+import org.eclipse.smarthome.core.types.Command;
+import org.openhab.binding.yeelight.internal.YeelightBindingConstants;
+import org.openhab.binding.yeelight.internal.lib.device.DeviceStatus;
+import org.openhab.binding.yeelight.internal.lib.enums.ActiveMode;
+
+/**
+ * The {@link YeelightCeilingWithNightHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author DragonNP - Initial contribution
+ */
+public class YeelightCeilingWithNightHandler extends YeelightHandlerBase {
+
+    public YeelightCeilingWithNightHandler(Thing thing) {
+        super(thing);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+        handleCommandHelper(channelUID, command, "Handle ceiling ambient light command");
+    }
```

Comment:
I don't think you need to override this. YeelightCeilingHandler already implements this for you, unless you want to fix the string instead.